applications/FluidDynamicsHydraulicsApplication/python_scripts/navier_stokes_two_fluid_hydraulic_fractional_solver.py

```diff
@@ -0,0 +1,644 @@
+
+# Import base class file
+from KratosMultiphysics.FluidDynamicsApplication.fluid_solver import FluidSolver
+
+
+from pathlib import Path
+
+def CreateSolver(model, custom_settings):
+    return NavierStokesTwoFluidsHydraulicFractionalSolver(model, custom_settings)
+
+class NavierStokesTwoFluidsHydraulicFractionalSolver(FluidSolver):
+
+    @classmethod
+    def GetDefaultParameters(cls):
+        ##settings string in json format
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "solver_type": "two_fluid_hydraulic",
+            "model_part_name": "",
+            "domain_size": -1,
+            "model_import_settings": {
+                "input_type": "mdpa",
+                "input_filename": "unknown_name",
+                "reorder": false
+            },
+            "material_import_settings": {
+                "materials_filename": ""
+            },
+            "maximum_iterations": 7,
+            "echo_level": 0,
+            "compute_reactions": false,
+            "analysis_type": "non_linear",
+            "reform_dofs_at_each_step": false,
+            "consider_periodic_conditions": false,
+            "relative_velocity_tolerance": 1e-3,
+            "absolute_velocity_tolerance": 1e-5,
+            "relative_pressure_tolerance": 1e-3,
+            "absolute_pressure_tolerance": 1e-5,
+            "linear_solver_settings"       : {
+                "solver_type"         : "amgcl"
+            },
+            "volume_model_part_name" : "volume_model_part",
+            "skin_parts": [""],
+            "assign_neighbour_elements_to_conditions": true,
+            "no_skin_parts":[""],
+            "time_stepping"                : {
+                "automatic_time_step" : true,
+                "CFL_number"          : 1,
+                "minimum_delta_time"  : 1e-2,
+                "maximum_delta_time"  : 1.0,
+                "time_step"           : 0.0
+            },
+            "move_mesh_flag": false,
+            "formulation": {
+                "dynamic_tau": 1.0,
+                "mass_source":true
+            },
+            "artificial_viscosity": false,
+            "artificial_visocosity_settings":{
+                "limiter_coefficient": 1000
+            },
+            "time_order": 2,
+            "time_scheme": "bdf2",
+            "fractional_splitting_settings":{
+                "element_type" : "ns_fractional_velocity_convection"
+             },
+            "levelset_convection_settings": {
+                "max_CFL" : 1.0,
+                "max_substeps" : 0,
+                "eulerian_error_compensation" : false,
+                "element_type" : "levelset_convection_supg",
+                "element_settings" : {
+                    "dynamic_tau" : 1.0,
+                    "tau_nodal":true
+                }
+            },
+            "outlet_inflow_limiter_settings":{
+                "apply_inflow_limiter": false,
+                "inflow_model_part_name":""
```

Comment:
Not sure if this belongs to this PR, but shouldn't this be a list of modelparts?